10/umbraco-cms/reference/configuration/contentsettings.md

```diff
@@ -59,6 +60,11 @@ In the root level section, that is those without a seperate sub section like Ima
 
 If greater control is required than available from the above, this setting can be used to store a list of file extensions. If provided, only files with these extensions can be uploaded via the backoffice.
 
+### Allowed media hosts
+
+By default only relative urls are allowed when getting URLs for resized images or thumbnails using the ImagesController.
+If you need absolute urls you will have to add the allowed hosted in this list. The value could be `["umbraco.com", "www.umbraco.com", "our.umbraco.com"]`.
```

Comment:
```suggestion
By default, only relative URLs are allowed when getting URLs for resized images or thumbnails using the ImagesController.
If you need absolute URLs you will have to add the allowed hosts to this list. The value could be `["umbraco.com", "www.umbraco.com", "our.umbraco.com"]`.
```

Reply:
All suggestions are find, we just need to ensure its fixed for both v10 and v11 version

Reply:
Will do 💪